README.md

```diff
@@ -23,6 +26,37 @@ This repository contains an SDK and Jupyter notebooks with examples to showcase
+1. Actively incorporates state-of-the-art experimental interpretability techniques, and allows for further expansion by researchers and data scientists 
+2. Creates a common API across the integrated libraries    
+3. Applies optimizations to make it feasible to run on real-world datasets at scale   
+4. Provides improvements such as the capability to "reverse the feature engineering pipeline" to provide users with feature importance values in terms of raw features
+5. Provides interactive and exploratory visualization to empower data scientists to get the most level of insights about their data
+# <a name="comparison"></a> 
+
+# Interpret vs. Interpret-Community
+
+
+Interpret-Community and its parent, Interpret, both serve as a one-stop-shop tool for researchers, machine learning engineers, software developers, data scientists, and business executives to get insights about the model predictions and why model made the prediction it did. The parent repository, Interpret, hosts the interpretability techniques that are widely accepted by research community and have been published in prestigous  publications and press (e.g., SHAP, EBM). Interpret-Community is an extension to Interpret and hosts experimental interpretability techniques that are yet to receive acceptance from the research community.            
+
+
+
+
+  # <a name="target"></a> 
+
+ # Target Audience
+1. Researchers: Interpret's extension hooks make it easy to extend and thus, researchers can easily add their interpretability solutions to the community repository and compare it to state-of-the-art interpretability techniques and other community solutions.
+
+2. Developers/Data Scientists: Having all of the state-of-the-art algorithms in one place makes it easy for data scientist to experiment with different interpretability techniques, and explain their model in a scalable and seamless manner. The set of rich interactive visualizations allow customers to train and deploy more transparent machine learning models instead of wasting time and effort on generating customized visualizations, addressing scalability issues by optimizing third-party interpretability techniques, and adopting/operationalizing interpretability solutions. 
+3. Business Executives: The core logic and visualizations are beneficial for raising awareness among those involved in developing AI applications, allow them to audit the output of models used by AI applications, and establish a strong governance framework around the use of AI applications.

```

Comment:
the output of models used by AI apps-> audit model predictions for potential bias and use this insight to help establish stronger governance capabilities.